conans/model/conf.py

```diff
@@ -231,14 +247,18 @@ def _get_boolean_value(value):
 
     def get(self, conf_name, default=None, check_type=None):
         """
-
         Get all the values of the given configuration name.
 
         :param conf_name: Name of the configuration.
         :param default: Default value in case of conf does not have the conf_name key.
         :param check_type: Check the conf type(value) is the same as the given by this param.
                            There are two default smart conversions for bool and str types.
         """
+        # Skipping this check only the user.* configurations
+        if USER_CONF_PATTERN.match(conf_name) is None and conf_name not in BUILT_IN_CONFS:
```

Comment:
Maybe just checking that it doesn't start with ``user.``, then it MUST exist in the dict.

Reply:
Like that must be a pattern, could we mark this comment as resolved? Because I think it's doing exactly what you're saying